config/jobs/kubernetes/sig-cloud-provider/azure/release-1.23.yaml

```diff
@@ -389,7 +389,7 @@ periodics:
       - -c
       - >-
         kubectl apply -f templates/addons/azurefile-role.yaml &&
-        cd ${GOPATH}/src/sigs.k8s.io/azurefile-csi-driver && ./deploy/install-driver.sh master local,snapshot,enable-avset &&
+        cd ${GOPATH}/src/sigs.k8s.io/azurefile-csi-driver && ./deploy/install-driver.sh master local &&
```

Comment:
@andyzhangx this change is the result of our recent discussion. Is this change O.K. for `capz-azure-file-vmss-1-23` job?

Reply:
lgtm

Reply:
thanks so much!